### PR TITLE
[HOPSWORKS-3010] add bigquery connector

### PR DESCRIPTION
**Pull request by DhananjayMukhedkar**
add bigquery connector

fix bigquery FK name

change key path to inode

test spark-che

add drop ddl and fixes

intial bigquery

### CODE REVIEW

File: files/default/sql/ddl/updates/2.6.0.sql [hunk 1]
```diff
@@ -134,3 +134,29 @@ SET `predictor_resources` = JSON_SET(`predictor_resources`, '$.requests', CAST(`
...
+    CONSTRAINT `fk_fs_storage_connector_bigq_keyfile` FOREIGN KEY (
+        `key_inode_pid`,
+        `key_inode_name`,
+        `key_partition_id`
+    ) REFERENCES `hops`.`hdfs_inodes` (`parent_id`, `name`, `partition_id`)
+) ENGINE = ndbcluster DEFAULT CHARSET = latin1 COLLATE = latin1_general_cs;
+
+ALTER TABLE `hopsworks`.`feature_store_connector`
+    ADD COLUMN `bigquery_id` INT,
+    ADD CONSTRAINT `fs_connector_bigquery_fk` FOREIGN KEY (`bigquery_id`) REFERENCES `hopsworks`.`feature_store_bigquery_connector` (`id`) ON DELETE CASCADE;
```
Comment (moritzmeister) [inline files/default/sql/ddl/updates/2.6.0.sql:160]: please add also `ON UPDATE NO ACTION` to be consistent with the create stable in the initial table definitions.

**Review by moritzmeister — CHANGES_REQUESTED** [review]
One small change

File: files/default/sql/ddl/updates/2.6.0.sql (hunk 1 above)
Comment (DhananjayMukhedkar) [reply files/default/sql/ddl/updates/2.6.0.sql]: added at missing places

File: files/default/sql/ddl/2.6.0__initial_tables.sql [hunk 2]
```diff
@@ -1810,6 +1810,27 @@ CREATE TABLE IF NOT EXISTS `feature_store_gcs_connector` (
...
+    `id`                      int AUTO_INCREMENT,
+    `key_inode_pid` BIGINT(20) NULL,
+    `key_inode_name` VARCHAR(255) NULL,
+    `key_partition_id` BIGINT(20) NULL,
+    `parent_project`          varchar(1000) NOT NULL,
+    `dataset`                 varchar(1000) NULL,
+    `query_table`             varchar(1000) NULL,
+    `query_project`           varchar(1000) NULL,
+    `materialization_dataset` varchar(1000) NULL,
+    `arguments`               varchar(2000) NULL,
```
Comment (DhananjayMukhedkar) [inline files/default/sql/ddl/2.6.0__initial_tables.sql:1825]: added arguments fields for key-value options

**Review by moritzmeister — APPROVED** [review]

**Pull request merged**